src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,17 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        try {
+            return numbers.stream()
+                    .map(w -> w.split(","))
+                    .flatMap(Arrays::stream)
+                    .mapToInt(Integer::parseInt)
+                    .filter(i -> i % 2 == 0)
+                    .min()
+                    .orElseThrow();
+        } catch (RuntimeException e) {
+            throw new RuntimeException("Can't get min value from list");
+        }
```

Comment:
```suggestion
            return numbers.stream()
                    .flatMap(n -> Arrays.stream(n.split(",")))
                    .mapToInt(Integer::parseInt)
                    .filter(i -> i % 2 == 0)
                    .min()
                    .orElseThrow( () -> throw new RuntimeException("Can't get min value from list"););
```

why do we need try-catch block here? We can combine map and flatMap in one flatMap